src/Payload/Options.php

```diff
@@ -11,6 +11,7 @@ public function __construct(
         public readonly ?bool $immutable = null,
         public readonly ?bool $forceAccount = null,
         public readonly ?ReturnUrl $returnUrl = null,
+        public readonly ?array $signers = null,
```

Comment:
Hi! Thanks for the PR. Could you make this `string ...$signers` instead or guarantee string type in some other way? 

And testing that this is properly serialized would be nice as well. Thanks in advance.